docs/framework/install/net-framework-not-found.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Troubleshooting 'This application could not be started'
+description: Learn what to do if you see a 'This application could not be started' dialog box.
+author: rpetrusha
+ms.author: ronpet
+ms.date: 09/05/2019
+---
+# Troubleshooting a 'This application could not be started' error message
+
+Applications that are developed for the .NET Framework typically require that a specific version of the .NET Framework be installed on your system. In some cases, you may attempt to run an application without either an installed version or the expected version of the .NET Framework present. This often produces an error dialog box like the following:
+
+![This application could not be started](media/app-could-not-be-started.png)
+
+This typically indicates one of the following conditions:
+
+- A .NET Framework installation on your system has become corrupted.
+
+- The version of the .NET Framework needed by your application cannot be detected.
+
+To address this issue so that you can run your application, do the following:
+
+1. Download the [.NET Framework Repair Tool (NetFxRepairTool.exe)](https://www.microsoft.com/en-us/download/details.aspx?id=30135). The tool runs automatically when the download completes.
```

Comment:
I think it would be better to remove `en-us` from the link.

Reply:
Thanks for noticing them, @Youssef1313. I've removed them.